include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -89,17 +89,21 @@ A_{3, 2}  A_{3, 3}  A_{3, 0}  A_{3, 1} ...   [phase 1] /
           if (isKDimInner) {
             const int numBanks = 32;
             const int bankBitWidth = 32;
+            const int SIMDWidth = 16;
 
             // number of inner dimension rows per one pattern repeat
-            int outerDimGranularity = mfmaEnc.getNonKDim();
             int typeBitWidth = eltTy.getIntOrFloatBitWidth();
             int innerDimLength = shape[order[0]];
             int elemsPerOneBanksRow = (numBanks * bankBitWidth) / typeBitWidth;
 
             int perPhase = std::max(1, elemsPerOneBanksRow / innerDimLength);
-            int maxPhase = outerDimGranularity / perPhase;
-            int vecSize = innerDimLength / maxPhase;
-            assert(vecSize > 0);
+            // Note: the following settings is customized to avoid
+            // **load** bank conflicts
+            // vecSize is set to k_base, which is the number of elements each
+            // workitem loads for one mfma instruction
+            // maxPhase is set to SIMDWidth / perPhase
+            int vecSize = dotOpEnc.getMMAv2kWidth();
```

Comment:
Yeah, it is misleading. But it turns out that `MMAv2kWidth` and `k_base` are the same. I also do not want to change `MMAv2kWidth` to something more generic, such as `kWidth`, since that will introduce a lot of conflicts when we do IFUs.

My original thought was to set `vecSize` to `k_base`, since this is the number of elements each workitem needs for **one** mfma. And based on that, I made `dotOperandEncoding` to carry that information. However, after a second thought I think it is not necessary since I want to try to extend the load vector length to be larger than `k_base`. So it is better to set `vecSize` independent of `k_base`.